.github/workflows/pull_requests.yml

```diff
@@ -22,6 +22,24 @@ jobs:
       with:
         configuration-path: .github/labeler-pr-needs-triage.yml
         repo-token: ${{ secrets.GITHUB_TOKEN }}
+  SizeLabeler:
+    runs-on: ubuntu-latest
+    steps:
+      # See also: https://github.com/CodelyTV/pr-size-labeler/pull/26
+      - name: Apply Size Label
+        uses: bflad/pr-size-labeler@7df62b12a176513631973abfe151d2b6213c3f12
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          xs_label: 'size/XS'
+          xs_max_size: '30'
+          s_label: 'size/S'
+          s_max_size: '60'
+          m_label: 'size/M'
+          m_max_size: '150'
+          l_label: 'size/L'
+          l_max_size: '300'
+          xl_label: 'size/XL'
```

Comment:
if we get a PR > `size/XL` (not that we really get that many), we'll need to manually label?